stestr/subunit_runner/program.py

```diff
@@ -102,11 +102,11 @@ def list_test(test):
     :return: A tuple of test ids that would run and error strings
         describing things that failed to import.
     """
-    unittest_import_strs = set([
+    unittest_import_strs = {
         'unittest2.loader.ModuleImportFailure.',
```

Comment:
can we remove this now ?

Reply:
I'm not sure, maybe. I'd have to test a testtools suite (and a direct unittest2 suite) with the stestr subunit runner to see if we can trigger import failures during discovery to see the mangled test id.I'm not sure how important it is though, this is to check for import errors during discovery and treat them as test errors (instead of trying to run them and erroring with an odd importerror). I don't think there is any harm in being slightly conservative and just leaving it in the set, there is no real penalty for having an extra entry in the set even if it never matches.

Reply:
Yeah, no big deal to keep it, thanks.